.travis.yml

```diff
@@ -1,26 +1,18 @@
-go_import_path: github.com/container-storage-interface/spec
-
 language: go
 
 go:
   - 1.8.x
 
 before_install:
-  - gem install gist
   - curl -LO https://github.com/google/protobuf/releases/download/v3.3.0/protoc-3.3.0-linux-x86_64.zip
   - unzip protoc-3.3.0-linux-x86_64.zip
   - chmod +x bin/protoc
-  - export PATH="$(pwd)/bin:$PATH"
 
 install:
   - go get -u github.com/golang/protobuf/proto
   - go get -u github.com/golang/protobuf/protoc-gen-go
   - go get -u google.golang.org/grpc
-  - CSI_SPEC_FILE=spec.md make csi.proto
 
 script:
-  - make csi.pb.go
-
-after_success:
```

Comment:
I really liked having the generated artifacts, can we keep these? (requires the earlier `gem install ...`)